extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyDotNames.java

```diff
@@ -44,6 +45,8 @@
             .createSimple(org.jboss.resteasy.annotations.jaxrs.HeaderParam.class.getName());
     public static final DotName RESTEASY_MATRIX_PARAM = DotName
             .createSimple(org.jboss.resteasy.annotations.jaxrs.MatrixParam.class.getName());
+    public static final DotName RESTEASY_SSE_ELEMENT_TYPE = DotName
+            .createSimple(org.jboss.resteasy.annotations.SseElementType.class.getName());
```

Comment:
Could you remove the `SSE_ELEMENT_TYPE` constant at the top of `ResteasyCommonProcessor` and only use the new one you created?